Bottom: 7f4c50882ec98a73e1d360210c7a6516c346f3e1
Top:    791df09bc5797df2df555ff05926fbb23434424a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 18:56:04 +0100

Refresh of add-cache-for-noholes-diced-ve

---

diff --git a/src/polygon.c b/src/polygon.c
index d7157ff..367c4eb 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1552,6 +1552,7 @@ r_NoHolesPolygonDicer (PLINE * p,
   pa->contours = p;
   if (!p->next)                 /* no holes */
     {
+      /* The callback now owns the contour */
       emit (p, user_data);
       pa->contours = NULL;
       poly_Free (&pa);
